Classes/Controller/EmbedController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+namespace KoninklijkeCollective\KoningInstagram\Controller;
+
+/**
+ * Controller: Embed
+ *
+ * @package KoninklijkeCollective\KoningInstagram\Controller
+ */
+class EmbedController extends \TYPO3\CMS\Extbase\Mvc\Controller\ActionController
+{
+    /**
+     * @var \KoninklijkeCollective\KoningInstagram\Service\InstagramService
+     * @inject
+     */
+    protected $instagramService;
+
+    /**
+     * @return void
+     */
+    public function showAction()
+    {
+        $code = $this->instagramService->getEmbedCode($this->settings['data']);
```

Comment:
Seems more legit to move this to a viewhelper so it can be used easily outside the action too